### ISSUE TITLE
Caching 404s

### ISSUE THREAD

**Issue by NickLydon** [filipw/AspNetWebApi-OutputCache#129]
Is it possible to cache responses with a 404 status code?

**Comment by kbaldyga** [comment]
Hi @NickLydon, did you manage to cache those?

**Comment by NickLydon** [comment]
@kbaldyga I don't think you can with the existing API. It is desirable though, as it would probably mitigate against denial of service and clients that have kept held of stale data.
